StatEnum.py

```diff
@@ -8,4 +8,17 @@ class Stat(Enum):
 	Spd = 4
 	Lck = 5
 	Def = 6
-	Res = 7
\ No newline at end of file
+	Res = 7
+
+def stat_dict(array):
+	# TODO: Add doc string
```

Comment:
"""
   stat_dict makes a dictionary mapping stats onto values for those stats

Args:
    array : a list of floats giving values of the form
        [HP, Str, Mag, Skl, SPd, Lck, Def, Res]

Returns:
    a dict<Stat, float> describing values for game stats
"""